storage/Devicegraph.cc

```diff
@@ -43,13 +44,15 @@ namespace storage
 
 
     WrongNumberOfParents::WrongNumberOfParents(size_t seen, size_t expected)
-	: Exception(sformat("wrong number of parents, seen '%zu', expected '%zu'", seen, expected))
+	: Exception(sformat("wrong number of parents, seen '%zu', expected '%zu'", seen, expected),
+		    Silencer::is_any_active() ? DEBUG : WARNING)
```

Comment:
Do you seriously want to distribute such a setting all over the place? Because that's what's going to happen if we tell the Silencer in 4223423 locations what log level to use.

Bad idea.